README.md

```diff
@@ -16,7 +16,7 @@ _Note: This plugin currently uses a development version of Eleventy which includ
 1. Install plugin using npm:
 
    ```
-   npm install eleventy-plugin-ghost --save
```

Comment:
Take or leave this one, but I tend to leave it off now since it's the default and newer versions of npm don't even document the `--save` flag.

Reply:
I didn't know this! I've been adding it out of habit and hadn't noticed. I'll go with it and if some people get stuck with it for some reason then I may put it back. Thanks!